docker/bacalhau-base/Dockerfile

```
@@ -1,14 +1,17 @@
 FROM ubuntu:24.04
 
+# Install essential tools and certificates
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)